src/components/implementation/no_interface/vkernel/vk_api.h

```diff
@@ -4,10 +4,10 @@
 #include "vk_types.h"
 
 /* extern functions */
-extern void vm_exit(void *);
 extern void vm_init(void *);
 extern void dom0_io_fn(void *);
 extern void vm_io_fn(void *);
+extern void *__inv_vkernel_serverfn(int a, int b, int c);
```

Comment:
I'm able to use it for my research and others are mainly using this as reference. I know and I understand that it has the potential to be a library but I don't know if it is going to take a lot of time to make it one. :( 
I'd work on that after the deadline.